### PR TITLE
Better visuals for signature help

### PR DESCRIPTION
**Pull request by yyoncho**
- use page-break-lines-mode to separate the sections

### CODE REVIEW

File: lsp-mode.el [hunk 1]
```diff
@@ -57,6 +57,7 @@
 (require 'minibuffer)
 (require 'yasnippet nil t)
 (require 'lsp-protocol)
+(require 'page-break-lines)
```
Comment (kiennq) [inline lsp-mode.el:60]: Is this built-in package? Else you have to put that one into package requires.

**Comment by kiennq** [comment]
Can you add some screenshot for this?
Do we need to use another package for this or can just walk away with insert bunches of `-` in place of `^L`?

File: lsp-mode.el (hunk 1 above)
Comment (ericdallo) [reply lsp-mode.el:60]: true, also on makefile for CI tests I think

Comment (yyoncho) [reply lsp-mode.el:60]: I thought that it is built-in, it is in melpa and built-in for spacemacs.  I will look for a solution

**Comment by yyoncho** [comment]
> Can you add some screenshot for this?
> Do we need to use another package for this or can just walk away with insert bunches of `-` in place of `^L`?

Screenshot: 
image


I don't want to go in the direction of inserting a bunch of `-` - in the end in order to look good we will have to reimplement the 
page-break-mode package.

**Comment by ericdallo** [comment]
Looks way better with this package indeed, maybe try to reimplements only the necessary?

**Comment by jcs090218** [comment]
Looks awesome! Can't wait to see this to merge! :)

**Review by ericdallo — APPROVED** [review]

**Review by jcs090218 — APPROVED** [review]

**Pull request merged**